client/src/main/resources/translations_en.properties

```diff
@@ -0,0 +1,8 @@
+choose_settings=Choose settings
+select_room=Select room
+create_room=Create room
+connect=Connect
+room_list=Rooms list
+player_name=Player name
+row_count=Row count
```

Comment:
plural / singular in row/columns texts - decide

Reply:
changed